packages/core/package.json

```diff
@@ -743,7 +743,7 @@
                     "type": "webview",
                     "id": "aws.amazonq.transformationHub",
                     "name": "Status",
-                    "when": "gumby.isTransformAvailable || gumby.isStopButtonAvailable"
+                    "when": "gumby.wasQCodeTransformationUsed"
```

Comment:
Simplified this logic a lot; Transformation Hub simply shows once the user clicks on Transform or does /transform action from chat.